src/CONST.ts

```diff
@@ -244,6 +244,76 @@ const CONST = {
         NEW_DOT_CATEGORIES: 'newDotCategories',
         NEW_DOT_TAGS: 'newDotTags',
         NEW_DOT_SAML: 'newDotSAML',
+        PDF_META_STORE: 'pdfMetaStore',
```

Comment:
Can you give a bit of context here? What is the source of all these new properties?

Reply:
@blazejkustra CONST.BETAS is used in [Beta type](https://github.com/Expensify/App/blob/main/src/types/onyx/Beta.ts#L4C1-L4C40)
There are two apiMocks that set betas, inside [openApp.ts](https://github.com/VickyStash/Expensify/blob/ts-migration/e2e-lib/src/libs/E2E/apiMocks/openApp.ts#L1453) and [signinUser.ts](https://github.com/VickyStash/Expensify/blob/ts-migration/e2e-lib/src/libs/E2E/apiMocks/signinUser.ts#L38)

So without these new properties, it gives an error like this:
![image](https://github.com/Expensify/App/assets/23176449/fbb87ee9-6e4e-4516-97c2-b22b07a46271)

Reply:
I am working on removing all useless ones, and this change made it 10x harder in this repo 😞

Reply:
@flodnv Why it made harder for you? Can you give a bit more context?

Reply:
Because now I have to check if all of these betas are used, which they probably are not... 

It would have been better to update the mocks to match the code instead of the opposite, which I think is probably almost always preferred (why mock something that does not exist / is not used?)

Reply:
Fixed here: https://github.com/Expensify/App/pull/31783